android/KMEA/app/src/main/assets/android-host.js

```diff
@@ -1,3 +1,10 @@
+var _debug = 1;
```

Comment:
```suggestion
var _debug = 0;
```

Let's turn off debug logging before merge 😁